Rakefile

```diff
@@ -20,5 +20,7 @@ require "yast/rake"
 
 Yast::Tasks.configuration do |conf|
   # The package does not live in the official YaST:Head OBS project
-  conf.obs_project = "home:aschnell:storage-redesign"
+  conf.obs_project = "YaST:storage-ng"
+  # Non-existent project to prevent accidental submission to Tumbleweed
+  conf.obs_sr_project = "NONE"
```

Comment:
Well, not sure which case is worst or better actually :wink: But none of them are desirable.